source/lambda/helper/python/kendraHelper.py

```diff
@@ -25,11 +25,78 @@ def indexDocument(self,
                       s3bucket,
                       s3key,
                       documentId,
+                      documentExtension,
                       tag = 'everybody'):
 
-
+    
+        # try to fetch the optional kendra policy file that may have been uploaded to s3
+        # along with the document
+        originalDocumentName = s3key[:-15].split('/')[3]
```

Comment:
I would prefer to avoid magic numbers in the code.
How about geting the doc name as 
s3Key.rsplit('/',1)[1].split('.')[0]